driver/src/main/java/org/neo4j/driver/internal/security/TLSSocketChannel.java

```diff
@@ -81,7 +82,7 @@ public static TLSSocketChannel create( ByteChannel channel, Logger logger, SSLEn
         }
         catch ( SSLHandshakeException e )
         {
-            throw new ClientException( "Failed to establish secured connection with the server: " + e.getMessage(), e );
+            throw new UnauthorizedException( "Failed to establish secured connection with the server: " + e.getMessage(), e );
```

Comment:
I would prefer to have fewer public errors. So how about change both of them to `SecurityException`/`AuthorizationException`?